item-spec/item-spec.md

```diff
@@ -36,15 +36,16 @@ incorporated.
-| properties | Properties Object                                                          | **REQUIRED.** A dictionary of additional metadata for the item.                                                                                                                                                                                                                                                                    |
-| links      | [Link Object]                                                              | **REQUIRED.** List of link objects to resources and related URLs. A link with the `rel` set to `self` is required.                                                                                                                                                                                                           |
-| assets     | Map<string, Asset Object>                                                  | **REQUIRED.** Dictionary of asset objects that can be downloaded, each with a unique key. Some pre-defined keys are listed in the chapter 'Asset types'.                                                                                                                                                                    |
+| bbox       | [number]                                                                   | **REQUIRED.** Bounding Box of the asset represented by this item. Specified in Longitude/Latitude based on EPSG:4326 - first two numbers are longitude and latitude of lower left corner, followed by longitude and latitude of upper right corner. This field enables more naive clients to easily index and search geospatially. Most software can easily generate them for footprints. STAC compliant APIs are required to compute intersection operations with the item's geometry field, not its bbox. |
+| properties | Properties Object                                                          | **REQUIRED.** A dictionary of additional metadata for the item. |
+| links      | [Link Object]                                                              | **REQUIRED.** List of link objects to resources and related URLs. A link with the `rel` set to `self` is required. |
+| assets     | Map<string, Asset Object>                                                  | **REQUIRED.** Dictionary of asset objects that can be downloaded, each with a unique key. Some pre-defined keys are listed in the chapter 'Asset types'. |
+| collection | string                                                                     | The `id` of the STAC Collection this Item references to (see `collection` relation type below). This field is *required* if such a relation type is present. This field provides an easy way for a user to search for any Items that belong in a specified Collection. |
```

Comment:
Here it says that this is a required field if the collection relation is present yet in the readme it states that this field is strongly recommended. These seem to be contradicting.

Reply:
Thanks for spotting this, I'll check what we prefer.

Reply:
If I get you correct, than you are referring to the collection rel type mentioned in the Item spec (strongly recommended) and to the collection field/property for items. It makes sense the way it is: It is strongly recommended to link to a collection. If a link to a collection is in the Item, it is required to add a collection field/property.

Reply:
Ah, you're right! Sorry about that.